python/ray/_private/worker.py

```diff
@@ -2641,12 +2641,12 @@ def put(
 @PublicAPI
 @client_mode_hook
 def wait(
-    object_refs: List["ray.ObjectRef"],
+    object_refs: List[ray.ObjectRef],
```

Comment:
@peytondmurray as per discussion on Slack, please try to revert these. If not possible, we'll ask for Core reviewers

Reply:
I was able to revert this.